### DISCUSSION TITLE
Connect via command line with a password containing special characters

### DISCUSSION THREAD

**Discussion by JDelSignore** [rustdesk/rustdesk#10088] [General]
### Bug Description

Connecting via command line with a password containing special characters (confirmed with '"' and '#') gives an error.

### How to Reproduce

Use the following command to reproduce: --connect 123456789 --password asdQWE123#

### Expected Behavior

I expected there would have a way to escape these special characters, but I tried everything I could think of and nothing worked.

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 11 -> Windows 11

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.3 -> 1.3.3

### Screenshots

![2024-11-28_12-21-45](https://github.com/user-attachments/assets/43f8220a-4de8-4625-8176-06eb398a0a36)

**Comment by JDelSignore** [comment]
Actually, it works when quoting the password, but only if the RustDesk client isn't already running on the local computer. Is that a normal limitation?

**Reply by c-michail** [reply]
It doesn't work with quotes on me
